compliance_checker/cf/cf.py

```diff
@@ -119,7 +119,6 @@ def __init__(self, options=None):
             "7.2": "§7.2 Cell Measures",
             "7.3": "§7.3 Cell Methods",
             "7.4": "§7.4 Climatological Statistics",
-            "7.5": "§7.5 Geometries",
```

Comment:
I'd like to see this go back in. It's documentary, and having a 7.5 implementation in the codebase but not adding it to this map seems misleading.

Reply:
The various new appendix entries follow the pattern of extending the respective CF checker classes.   This change follows the convention previously established by the checker classes and also makes clear when a new section has been introduced.  CF < 1.8 does not have a check for section 7.5 and this makes that clear.

Reply:
OK, gotcha.

Reply:
Also, if a section is ever renamed or moved in a release of CF (to my knowledge this hasn't occurred yet), we will have to put the updated dict entry in the new class anyhow.